src/components/CountryDetails.jsx

```diff
@@ -0,0 +1,54 @@
+import { useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import Navigation from './Navigation';
+import TopBar from './TopBar';
+import {
+  detailsSelector,
+  fetchCountryDetails,
+} from '../redux/details/detailsSlice';
+import '../styles/CountryDetails.css';
+import DetailsTable from './DetailsTable';
+
+export default function CountryDetails() {
+  const params = useParams();
+  const dispatch = useDispatch();
+
+  const { countryName } = params;
+  const { countryDetails, isLoading, error } = useSelector(detailsSelector);
+
+  useEffect(() => {
+    dispatch(fetchCountryDetails(countryName));
+  }, [countryName, dispatch]);
+  return (
+    <>
+      <div className="details-container">
+        <Navigation>
+          <div className="header-title">
+            <h5>COVID-19 Cases</h5>
+          </div>
+        </Navigation>
+        {isLoading && <p className="loading">Loading...</p>}
+        {!!Object.keys(countryDetails).length && !isLoading && !error && (
+          <>
+            <TopBar
+              src={countryDetails?.countryInfo?.flag}
+              title={countryDetails?.country}
+              population={countryDetails?.population?.toLocaleString()}
+            />
+            <h4>
+              {countryName.toUpperCase()}
+              : COVID-19 current stats
+            </h4>
```

Comment:
[REQUIRED] Kindly reduce the space at the left and right parts at the top section of the details page so that your design can be more similar to the template.

![image](https://github.com/UbdaNam/africa-covid-19-tracker/assets/62938735/9e07c3eb-2102-4e8b-a2b9-12d0f0b84310)